### PR TITLE
feat(models): add BSyncr sample data example

### PR DESCRIPTION
**Pull request by laurenliz22**
#### Any background context you want to provide?
This update now includes the ability to use BSyncr analysis one of the sample data records.

#### What's this PR do?
This update now includes the ability to use BSyncr analysis for pm_parent_property_id = 5766973.  To make this work the record needed to have longitude/latitude and meter data.

#### How should this be manually tested?
Test that the record is able to produce BSyncr analysis in SEED's Platform

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/projects/14#card-58380230

### CODE REVIEW

**Review by macintoshpie — CHANGES_REQUESTED** [review]
One comment about a missing file. Also maybe we could update the modal to indicate you can run a BSyncr analysis on the property?
Screen Shot 2021-06-02 at 12 58 03 PM

Once the file get's added I can do a full test with BSyncr

File: seed/views/v3/organizations.py [hunk 1]
```diff
@@ -1504,7 +1504,7 @@ def insert_sample_data(self, request, pk=None):
         import_record = ImportRecord.objects.create(name='Auto-Populate', super_organization=org)
 
         # Interval Data
-        filename = 'example-pm-monthly-meter-usage.xlsx'
+        filename = 'PM Meter Data 12.xlsx'
```
[the remark this answers is not in the document]
Comment (laurenliz22) [reply seed/views/v3/organizations.py:1507]: @macintoshpie forgot we changed that, should be added now.

**Review by macintoshpie — COMMENTED** [review]
Tested locally with BSyncr and it ran successfully 👍 

Could you add the notes to the Modal about being able to run BSyncr analysis on that property? Once that's done this is ready to merge

**Comment by laurenliz22** [comment]
@updated Modal about being able to run BSyncr analysis on property

**Review by macintoshpie — APPROVED** [review]
Nice 💥

**Comment by macintoshpie** [comment]
@laurenliz22 were you going to add anything else to this PR or can I merge it?

**Comment by laurenliz22** [comment]
> @laurenliz22 were you going to add anything else to this PR or can I merge it?

@macintoshpie this can be merged, nothing more to add - thank you!

**Pull request merged**